frontend/src/components/Session/ClientAvatar.module.css

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.avatar {
+    object-fit: cover;
+    overflow: hidden;
+    aspect-ratio: 1 / 1;
+    width: var(--mas-avatar-size);
+    border-radius: 50%;
```

Comment:
I'm not sure about that. Works well for our logos, but in other places (like the consent screen), we're not making logo rounds. I'll leave it like this for now, and we'll probably iterate on this later